addons/sys_core/fnc_updateSelf.sqf

```diff
@@ -16,12 +16,12 @@
  */
 
 // ref bug: http://feedback.arma3.com/view.php?id=15580
-private _projectPos = ATLtoASL positionCameraToWorld [0, 0, 0];
+private _projectPos = AGLtoASL positionCameraToWorld [0, 0, 0];
 
-if (EGVAR(sys_zeus,zeusCommunicateViaCamera) && {call FUNC(inZeus)}) then {
+if (EGVAR(sys_zeus,zeusCommunicateViaCamera) && FUNC(inZeus)) then {
     ACRE_LISTENER_DIR = eyeDirection player;
 } else {
-    ACRE_LISTENER_DIR = (ATLtoASL positionCameraToWorld [0, 0, 1]) vectorDiff _projectPos;
+    ACRE_LISTENER_DIR = AGLtoASL positionCameraToWorld [0, 0, 1] vectorDiff _projectPos;
```

Comment:
This helps increasing the readability of the order of operands. MISRA based rule to the SQF rescue...

```suggestion
    ACRE_LISTENER_DIR = (AGLtoASL positionCameraToWorld [0, 0, 1]) vectorDiff _projectPos;
```